core/modules/user/user.admin.inc

```diff
@@ -425,6 +425,18 @@ function user_login_settings($form, &$form_state) {
       '#description' => t('Show plain links to these pages, and hide all layout regions, such as headers, footers, and sidebars.'),
     ),
   );
+  $form['user_login_appearance_setting']['login_admin_theme'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Use the administration theme for simplified login pages'),
+    '#default_value' => $core_config->get('login_admin_theme'),
+    '#config' => 'system.core',
+    '#description' => t('If disabled, the default front-end theme will be used. This setting only has an effect when the "Show simplified page" is set.'),
```

Comment:
Since this is a dependent form element, it should be indented:

```suggestion
    '#description' => t('If disabled, the default front-end theme will be used. This setting only has an effect when the "Show simplified page" is set.'),
    '#indentation' => 1,
```